definitions/npm/jest_v22.x.x/flow_v0.39.x-/jest_v22.x.x.js

```diff
@@ -123,14 +123,16 @@ type EnzymeMatchersType = {
   toBeChecked(): void,
   toBeDisabled(): void,
   toBeEmpty(): void,
+  toBeEmptyRender(): void,
   toBePresent(): void,
   toContainReact(element: React$Element<any>): void,
+  toExist(): void,
   toHaveClassName(className: string): void,
   toHaveHTML(html: string): void,
-  toHaveProp(propKey: string, propValue?: any): void,
+  toHaveProp: ((propKey: string, propValue?: any) => void) & ((props: Object) => void),
```

Comment:
You can probably change this to `toHaveProp(propKey: string | Object, propValue?: any): void,` to match the changes in https://github.com/flowtype/flow-typed/pull/1932/files

Reply:
I did that initially, but this version yells at you if you both provide an object as the first argument and a string as the second. I'll change it to match my changes in the other file if that's okay with you :)